releases/release-1.31/release-notes/maps/pr-124465-map.yaml

```
@@ -0,0 +1,6 @@
+pr: 124465
+releasenote:
+  text: 'kubeadm: switch to using the new etcd endpoints introduced in 3.5.11 - /livez
```

```suggestion
  text: 'kubeadm: Switched to using the new etcd endpoints introduced in 3.5.11 - /livez
```

https://github.com/kubernetes/sig-release/pull/2542#discussion_r1653312974